plugins/chat/spec/system/navigation_spec.rb

```diff
@@ -1,22 +1,26 @@
 # frozen_string_literal: true
 
 RSpec.describe "Navigation", type: :system, js: true do
-  fab!(:user) { Fabricate(:user) }
+  fab!(:category) { Fabricate(:category) }
+  fab!(:topic) { Fabricate(:topic) }
+  fab!(:post) { Fabricate(:post, topic: topic) }
+  fab!(:user) { Fabricate(:admin) }
   fab!(:category_channel) { Fabricate(:category_channel) }
   fab!(:message) { Fabricate(:chat_message, chat_channel: category_channel) }
 
   before do
+    # ensures we have one valid registered admin
+    user.activate
```

Comment:
This is interesting, what happens if you don't activate?

Reply:
otherwise it will show the starting page of discourse "register an admin" we should probably make it a default, I tried to have it in the fabricator but it wasn't working, didn't want to go in the rabbit hole for now

Reply:
Ah yes makes sense, I guess this also plays into making some preloaded data fixtures for system specs too so we are always sure to have e.g. 1 admin user, 5 regular users etc. that are reset to starting state between every test.